CMakeLists.txt

```diff
@@ -10,12 +10,20 @@ SET(${PROJECT_NAME}_VERSION_PATCH 0)
 SET(${PROJECT_NAME}_VERSION "${${PROJECT_NAME}_VERSION_MAJOR}.${${PROJECT_NAME}_VERSION_MINOR}.${${PROJECT_NAME}_VERSION_PATCH}")
 
 #--- Definitely need EDM4HEP data model library
-find_package(EDM4HEP)
+find_package(EDM4HEP REQUIRED)
+
+#--- Also need ROOT to use some of the useful physics libraries
+find_package(ROOT REQUIRED COMPONENTS Core Physics)
 
 #--- Define basic build settings -----------------------------------------------
 # - Use GNU-style hierarchy for installing build products
 include(GNUInstallDirs)
 
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "")
+if (NOT CMAKE_CXX_STANDARD MATCHES "17")
```

Comment:
We wanted to do a test build with a newer compiler and `cxxstd=20`, so maybe you can already add 20

Reply:
Done.